CAIPs/caip-222.md

```diff
@@ -45,7 +46,7 @@ The application would interface with a provider to make request as follows:
   "params": {
       "cacaov": string,
       "type": string,
-      "chainId": string,
+      "chains": string[],
```

Comment:
should this be optional and the wallet can just respond with any of those it supports?

Reply:
The rationale is that the dapp would request signatures for all chains included in the `chains` parameter and the wallet would respond with signatures for those it supports as outlined [here](https://github.com/ChainAgnostic/CAIPs/blob/7087562ed1f3a49e62ca6eb739bc32b67961d1bc/CAIPs/caip-222.md#not-supported-chains)
>The wallet SHOULD ignore all unsupported chains and SHOULD NOT auto reject the request if there are supported chains to sign without explicit user rejection.

In v1, `chainId` is required parameter so it makes sense to preserve that to an extended.
Additionally, setting `chains` to explicitely empty `[]` array is valid and kinda allows to not set any chains already.